dump-pe/main.cpp

```diff
@@ -227,17 +227,20 @@ int printSymbols(void *N,
 
 int printRsrc(void *N, resource r) {
   static_cast<void>(N);
+  std::basic_string<char> name(r.name_str.begin(), r.name_str.end());
```

Comment:
In Windows, this doesn't convert from UTF-16 to UTF-8. It actually just breaks the each `char16_t` into 2 `char`s. I'd expect this is the same behavior on Linux.